protos/datamodel/infrastructure.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "datamodel/common.proto";
+import "datamodel/costs.proto";
+import "datamodel/nodes.proto";
+import "datamodel/compute.proto";
+import "datamodel/storage.proto";
+import "datamodel/networking.proto";
+import "datamodel/pods.proto";
+import "datamodel/workloads.proto";
+
+package datamodel.v2;
+option go_package = "github.com/opencost/opencost/core/pkg/model/pb;pb";
+
+// Cluster represents the top-level container following normalized approach
+// To serve as a wrapper for all resource collections
+message Cluster {
+  // Identification
+  string ID = 1;
```

Comment:
Remind me, do we know where this `ID` will come from? We need it, anyways. Just curious, as Kubernetes does not assign IDs to clusters.